.travis.yml

```diff
@@ -0,0 +1,37 @@
+dist: trusty
+sudo: true
```

Comment:
should be `sudo: required` according to Travis docs (yeah it may still work with true but better stick to the docs).

In fact, I think we don't even need `sudo`, we can install mono/libgdiplus into a writable prefix like /tmp and I the other packages should be available via https://docs.travis-ci.com/user/installing-dependencies/#Installing-Packages-with-the-APT-Addon